Index: scilab-5.5.2/m4/java.m4
===================================================================
--- scilab-5.5.2.orig/m4/java.m4
+++ scilab-5.5.2/m4/java.m4
@@ -481,7 +481,10 @@ AC_DEFUN([AC_JAVA_JNI_LIBS], [
         powerpc)
           machine=ppc
           ;;
-        ppc64|ppc64le)
+        ppc64le)
+          machine=ppc64le
+          ;;
+        ppc64)
           machine=ppc64
           ;;
         armv*)
Index: scilab-5.5.2/bin/scilab
===================================================================
--- scilab-5.5.2.orig/bin/scilab
+++ scilab-5.5.2/bin/scilab
@@ -528,9 +528,12 @@ detect_java_vm() {
                         "ppc"|"powerpc")
                             proc="ppc"
                             ;;
-                        "ppc64"|"ppc64el")
+                        "ppc64")
                             proc="ppc64"
                             ;;
+                        "ppc64le")
+                            proc="ppc64le"
+                            ;;
                         "s390")
                             proc="s390"
                             ;;
